plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -204,6 +206,11 @@ def sweep(self, arguments: List[str]) -> None:
             OmegaConf.create(results_to_serialize),
             f"{self.sweep_dir}/optimization_results.yaml",
         )
+
+        metadata_to_serialize = {"sweep_counter": sweep_counter}
+        OmegaConf.save(
+            OmegaConf.create(metadata_to_serialize), f"{self.sweep_dir}/metadata.yaml",
+        )
```

Comment:
Why is this interesting enough to create a new file for?

Reply:
`sweep_counter` counts how many times the launcher was called. This is used in the tests to make sure the launcher was called more than once.

Reply:
seems seems like a very hacky way to pass this information.
You can use several mode direct approaches:
1. mock the launcher of create a dummy launcher and have it save that.
2. somehow change the standard metadata stored by the launcher to add launcher_batch_num to the stored output in the job output dir.